src/mame/drivers/viper.cpp

```diff
@@ -134,13 +134,13 @@ Game ID  Year    Game
 GK922    2000    Code One Dispatch
 G????    2001    ParaParaParadise 2nd Mix
 GM941    2001    GTI Club 2
-G?A00    2001    Police 911 (USA) / Police 24/7 (World) / Keisatsukan Shinjuku 24ji (Japan)
+G?A00    2001    Police 911 (USA) / Police 24/7 (World) / The Keisatsukan: Shinjuku 24-ji (Asia/Japan/Korea)
 GKA13    2001    Silent Scope EX (USA/World) / Sogeki (Japan)
 G?A29    2001    Mocap Boxing
 G?A30    2002    Tsurugi
 GMA41    2001    Thrill Drive 2
 G?A45    2001    Boxing Mania
-G*B11    2001    Police 911 2 (USA) / Police 24/7 2 (World) / Keisatsukan Shinjuku 24ji 2 (Japan)
+G*B11    2001    Police 911 2 (USA) / Police 24/7 2 (World) / The Keisatsukan 2: Senkoku Taitaiseki Special (Japan)
 G?B33    2001    Mocap Golf
```

Comment:
I think Vas copied the transliteration from Wikipedia? There are many errors on the English page apparently; it was definitely not done by someone who speaks Japanese.

Anyway, the reading here should be "Zenkoku Daitsuiseki" instead of "Senkoku Taitaiseki".

Reply:
The Daitsuiseki part is it Dai Tsuiseki or Daitsuiseki?

Reply:
You'll find there's no consistent answer for such questions when it comes to English transliterations of languages that don't use spaces. For instance, you can find many occurrences of "Daisakusen" vs "Dai Sakusen" in the wild. MAME seems to almost exclusively use the former, so probably "Daitsuiseki" would be best. FWIW, here's a real world occurrence: https://en.wikipedia.org/wiki/Daitsuiseki.

Reply:
Also Google Translate is giving me 24-Ji with an uppercase J.

Reply:
If no answer I'm going uppercase J then.

Reply:
As a counter suffix lowercase looks more natural to me and is more consistent with existing descriptions in the software lists (and somewhat with machine descriptions, though there are few instances of counters in titles there). I'd leave it lowercase unless @cuavas tells us otherwise : )

Reply:
Too late. Well, leave it uppercase unless @cuavas tells us otherwise then :)

Reply:
Nevermind just lowercased it.  And fixed the typo in the sequel title.

Reply:
Nevermind just lowercased it.  And fixed the typo in the sequel title.

Reply:
>I think Vas copied the transliteration from Wikipedia?

Yes, I copied the transliteration from some site without actually reading it because I was in a hurry.  Sorry, it’s obviously wrong.

Reply:
It's all good Vas.

Reply:
It's okay Vas.  Got the translation mostly to completely right several days ago with 0kmg's help.